components/tab/theme.css

```diff
@@ -1,5 +1,16 @@
 @import '@teamleader/ui-colors';
 @import '@teamleader/ui-animations';
+@import '@teamleader/ui-utilities';
+
+.tab-group {
+  & > * {
+    margin-left: var(--spacer-small);
+  }
+
+  & > *:last-child {
+    margin-right: var(--spacer-small);
+  }
+}
```

Comment:
You can nest the last-child into the other selector.

```css
.tab-group {
   & > * {
     margin-left: var(--spacer-small);

     &:last-child {
       margin-right: var(--spacer-small);
     }
   }
 }
```

Reply:
Nice. Should've seen that it was 2 times the same.

 :ok: by @InstaK on 10/01/2018 11:51:21